datetime/src/main/java/com/fasterxml/jackson/datatype/jsr310/deser/MonthEnumDeserializerModifier.java

```diff
@@ -0,0 +1,60 @@
+package com.fasterxml.jackson.datatype.jsr310.deser;
+
+import java.io.IOException;
+import java.time.Month;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.util.JacksonFeatureSet;
+import com.fasterxml.jackson.databind.BeanDescription;
+import com.fasterxml.jackson.databind.DeserializationConfig;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.deser.BeanDeserializerModifier;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeFeature;
+
+
+public class MonthEnumDeserializerModifier extends BeanDeserializerModifier {
```

Comment:
Typically modules have at most one such modifier, not type specific. So maybe name as `JavaTimeDeserializerModifier` to allow future expansion.